relay-replays/src/recording.rs

```diff
@@ -68,31 +68,72 @@ impl From<serde_json::Error> for ParseRecordingError {
     }
 }
 
-static STRING_STATE: Lazy<ProcessingState> = Lazy::new(|| {
-    ProcessingState::root().enter_static(
-        "",
-        Some(Cow::Owned(FieldAttrs::new().pii(Pii::True))),
-        Some(ValueType::String),
-    )
+/// Paths to fields on which datascrubbing rules should be applied.
+///
+/// This is equivalent to marking a field as `pii = true` in an `Annotated` schema.
+static PII_FIELDS: Lazy<[Vec<&str>; 2]> = Lazy::new(|| {
```

Comment:
Maybe it would be better to move this to the top of the file and put just after imports? 

Also does it make sense to use `Vec` here?  Maybe just switch to another array instead for now? Since we have the same number of parts in the arrays, like
```suggestion
static PII_FIELDS: Lazy<[[&str;3]; 2]> = Lazy::new(|| {
```

Reply:
Moved it to the top. I'd like to keep them `Vec`s because the path length varies, it's a coincidence that they both have length 3 at the moment.